openapi/v2/core-data.yaml

```diff
@@ -483,6 +501,48 @@ paths:
               type: array
               items:
                 $ref: '#/components/schemas/AddEventRequest'
+            example:
+              - event:
+                  deviceName: device-002
+                  id: d5471d59-2810-419a-8744-18eb8fa03465
+                  origin: 1471806366619
```

Comment:
it is millisecond, not nanosecond.  origin is in nanosecond, and others timestamp nanosecond
please use **1602168089665565300** for all the origin